p2p.go

```diff
@@ -320,7 +320,7 @@ func NewHost(ctx context.Context, options ...Option) (*Host, error) {
 	if err != nil {
 		return nil, err
 	}
-	kad, err := dht.New(ctx, host, dht.ProtocolPrefix(ProtocolDHT))
+	kad, err := dht.New(ctx, host, dht.ProtocolPrefix(ProtocolDHT), dht.Mode(dht.ModeServer))
```

Comment:
in the default mode, node only passively listen
that's why i saw bootnode has all delegates as peers (all delegates connect to him)
but delegate only has bootnode (he only connects to bootnode)